.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Do we need .vs folder? If not, can we add to gitignore?

Reply:
No we don't need it.